Sources/Model/TestflightProgram.swift

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import Foundation
+
+/// Aggregated model data representing the a Testflight Beta Program (Apps, Testers and Groups)
+public struct TestflightProgram {
```

Comment:
This looks good now. Will these data be enough for our following comparison logic 🤔 ?

Reply:
What I am worried the most is `BetaTester` coz it doesn't have `var id`

Reply:
We may need to add it in the PR that begins to form those comparisons. Better to leave it until it's required